code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -285,12 +280,6 @@
 			M.emote("scream")
 	..()
 
-
-/datum/reagent/medicine/styptic_powder/on_mob_life(mob/living/M)
```

Comment:
Um, does this just remove Styptic Powder's effects without removing it from the game?

Reply:
no this makes it work like synthflesh does-no effect on metabolization, but it does heal on application instead.